.github/workflows/rails_main_testing.yml

```diff
@@ -0,0 +1,29 @@
+name: Test against Rails main
+
+on:
+  push:
+  schedule:
+    - cron: "0 0 * * *" # Run every day at 00:00 UTC
```

Comment:
maybe weekly, lots of overhead otherwise ...

Reply:
@zendesk/ruby-core do we want to revisit daily testing with Rails main now that we test so many gems every day or do we want to keep the quick feedback?